lib/recording/RecordingEntrySession.js

```diff
@@ -283,16 +284,22 @@ class RecordingEntrySession {
         this.logger.debug("About to update chunks: %s", chunksNameChained)
         return Q.allSettled(this.playlistGenerator.update(Mp4Files))
             .then((results)=> {
-                if (results[0].state == 'rejected') {
-                    this.logger.error("LinkAndUpdateJson Failed to update chunks: %s ", ErrorUtils.error2string(results[0].reason))
-                }
-                else {
-                    this.logger.debug("LinkAndUpdateJson, Successfully update: %s", chunksNameChained)
-                }
+                this.logger.debug("About to parse result of chunks: %s", chunksNameChained)
+                this.parseUpdateJsonResult(results[1].value)
                 return Q.resolve(chunksNameChained)
             })
     }
 
+    parseUpdateJsonResult(result){
```

Comment:
what are you trying to achieve?
in 1.2 there is code change: instead of returning array of removed chunks from update an event is raised.
so basically this code is obsolete.